### ISSUE TITLE
Train process

### ISSUE THREAD

**Issue by InstantWindy** [jytime/Mask_RCNN_Pytorch#9]
Hi ! Can you explain your training process? In other words, which file should be run for training, which file should be tested? Thanks !

**Comment by jytime** [comment]
hi

I am not sure if I understand your question.

The mode (training or testing) is determined by the flag _args.command_. Taking coco.py for an example,  if _args.command == "evaluate"_, the file will test something. This is determined by what you type in the terminal.

**Comment by InstantWindy** [comment]
No, I mean the training sequence, first pre-training in imagnet and then training on the coco dataset?

**Comment by jytime** [comment]
there is already a model pre-trained on Imagenet

**Comment by InstantWindy** [comment]
So, does it only need to be trained on the coco dataset? Thanks!

**Comment by jytime** [comment]
actually in this implementation we already have models trained our imagenet and coco, thanks to [lasseha](https://github.com/multimodallearning/pytorch-mask-rcnn)
what we do is to implement several features and train that on the SYNTHIA dataset

**Comment by InstantWindy** [comment]
Thanks!
